### PR TITLE
Bump http-types to 0.0.13

### PR DESCRIPTION
**Pull request by fornwall**

### CODE REVIEW

File: meeshkan/server/server/callbacks.py [hunk 1]
```diff
@@ -80,16 +80,7 @@ def _map_results(self, response, result, format, response_type):
...
-            sink = StringIO()
-            #### there is a bug where sometimes, for unclear reasons,
-            #### the request body is sometimes written as a byte string
-            #### instead of a string. we hackishly fix here, but note that
-            #### the bug exists. would be good to know why
-            request = replace(request, body=str(request.body))
-            HttpExchangeWriter(sink).write(HttpExchange(request=request, response=response))
-            sink.seek(0)
-            res = json.loads('\n'.join([x for x in sink]))
-            out = callback(res['request'], res['response'], storage_manager.default)
```
Comment (fornwall) [inline meeshkan/server/server/callbacks.py]: Could this bug have been fixed by https://github.com/meeshkan/meeshkan/commit/36d5f4597637170d7785e42132adf2b129645e09#diff-cc24a3db2500505592b6c0ea4f93ec18L65 ?

File: setup.py [hunk 2]
```diff
@@ -29,7 +29,7 @@
     'openapi-typed_2>=0.0.2',
     'typeguard>=2.7.0',
     'genson',
-    'http-types==0.0.11',
+    'http-types==0.0.13',
```
Comment (ksaaskil) [inline setup.py]: I would be brave here and set `http-types>=0.0.13,<0.1.0` and try and bump the version when adding breaking changes from now on.

File: meeshkan/server/server/callbacks.py [hunk 3]
```diff
@@ -80,16 +80,7 @@ def _map_results(self, response, result, format, response_type):
...
+            out = callback(asdict(request), asdict(response), storage_manager.default)
```
Comment (ksaaskil) [inline meeshkan/server/server/callbacks.py]: Is there some reason to convert to dictionary here?

Comment (ksaaskil) [reply meeshkan/server/server/callbacks.py]: Not related to this PR of course, just wondering in general why it's done. @aby2s

File: setup.py (hunk 2 above)
Comment (fornwall) [reply setup.py]: Makes sense, did that now!

**Pull request merged**